rustls/src/client/handy.rs

```diff
@@ -241,6 +241,32 @@ impl client::ResolvesClientCert for AlwaysResolvesClientCert {
     }
 }
 
+#[derive(Debug)]
+/// An exemplar `ResolvesClientCert` implementation that always resolves to a single raw public key.
+pub struct AlwaysResolvesClientRawPublicKeys(pub Arc<sign::CertifiedKey>);
```

Comment:
I think I'd prefer to keep the field private and a have simple `new()` constructor instead, to make it a little easier to potentially evolve the API if necessary.

(Nit: please put comments before attributes.)